backend/src/activitypub/objects/index.ts

```diff
@@ -179,3 +181,71 @@ WHERE objects.${key}=?
 		originalObjectId: result.original_object_id,
 	} as Object
 }
+
+/** Is the given `value` an ActivityPub Object? */
```

Comment:
is this comment beneficial? (also in the function there isn't anything ActivityPub specific, so I feel like the comment is a bit off?)

Reply:
This file is all about ActivityPub Objects.
I appreciate that the implementation is a bit thin. I originally had more checks in place but it appears that it is possible to create AP Objects with almost no properties!
The description is accurate for its usage. It should only be used when checking whether a value is an AP Object, in order to narrow the type.
Removing the description might confuse people into using this function in non-supported scenarios, which would then break if we change or tighten up the implementation.

Reply:
ok thanks for the clarification, I'm fine with the comment 🙂